.cargo/config.toml

```diff
@@ -3,5 +3,5 @@
 ck = "check --workspace --all-targets --all-features --locked"
 lint = "clippy --workspace --all-targets --all-features"
 codecov = "llvm-cov nextest --workspace --ignore-filename-regex tasks"
-benchmark = "run -p pacquet_benchmark --release --"
-benchmark-install-against-revisions = "run --bin=benchmark-install-against-revisions --"
+micro-benchmark = "run -p pacquet_micro_benchmark --release --"
```

Comment:
Since we are aiming for non-generic commands in here, I recommend removing benchmarks from cargo config and keep it only on justfile.